src/alert-action.js

```diff
@@ -0,0 +1,50 @@
+const common = require('./wdio-common')
+
+module.exports = function(RED) {
+
+    function alertAction(config) {
+        RED.nodes.createNode(this, config)
+        const node = this        
+        common.clearStatus(node)
+
+        node.on('input', async (msg) => {
+            try {
+                let browser = await common.getBrowser(node.context())
+               
+                let value = config.sendText || msg.value
+                
+                let isAlertPresent = async () => {
```

Comment:
This is an independent function so should be moved outside the `alertAction` function. Could be declared before or after `alertAction`, something like `async function isAlertPresent() {`

Reply:
onst isAlertPresent = async (browser) => {
    try{
        await browser.getAlertText()
        return true
    }
    catch(e){
        return false
    } 
}

module.exports = function(RED) {

    function alertAction(config) {
        RED.nodes.createNode(this, config)
        const node = this        
        common.clearStatus(node)
``

Reply:
Tested with latest changes.